### ISSUE TITLE
same code working on MAC but not windows

### ISSUE THREAD

**Issue by feranf** [senecajs/senecajs.org#73]
Hi,
I'm running the following code in MAC - works OK.
**When running it on windows 7 - the following error occurs..**
```javascript
function color() {
  this.add( 'color:red', function(args,done){
    done(null, {hex:'#FF0000'});
  })
}


var seneca = require('seneca')


seneca()
  .use(color)
  .listen({type:'tcp'})

seneca()
  .client({type:'tcp'})
  .act('color:red')

// node readme-color-tcp.js --seneca.log=plugin:transport,level:INFO --seneca.log=type:act,regex:color:red
```
* same for http transport
* firewall was disabled...

**The Error:**
2015-10-15T09:14:46.808Z s3fx6o3a4jgi/1444900486776/7384/- DEBUG        act     remote$         IN  m9bqsfq0d1ms/mj7vu98io1jj
        {color:red}     ENTRY   CLIENT  -       -       -
2015-10-15T09:14:57.946Z s3fx6o3a4jgi/1444900486776/7384/- ERROR        act     remote$         OUT     -       11135   {color:r
ed}     ENTRY   CLIENT  -       seneca: Action  failed: [TIMEOUT].      act_execute     {id:m9bqsfq0d1ms/mj7vu98io1jj,gate:false
,ungate:false,desc:,plugin:{name:remote$,tag:null},time:{start:14449004 Error: [TIMEOUT]
    at null._onTimeout (C:\seneca\node_modules\seneca\node_modules\gate-executor\gate-executor.js:95:
21)
    at Timer.listOnTimeout [as ontimeout] (timers.js:112:15)

Thanks,

**Comment by maxired** [comment]
@feranf , are you using the last version (0.7.1) of seneca  ?

**Comment by feranf** [comment]
@maxired yes

**Comment by maxired** [comment]
@feranf, using node 4.2.1 and seneca 0.7.1, it works for me on windows 8.1
I had similar issue before I unblock node.js from windows firewall, so you might want to check your firewall again.

Moreover, except if you found this code on seneca.org website, you probably want to post this kind of issue on https://github.com/rjrodger/seneca

**Comment by mcdonnelldean** [comment]
@feranf If you double check your firewall (and do a restart) and it's still not working, could you follow this link to post a new issue on the http / tcp transport modules issues page:

https://github.com/rjrodger/seneca-transport/issues/new

We don't have a huge amount of Windows users on different versions so if it turns out we have a problem I'd like to look at getting it resolved for you :D 

@maxired Do you have Windows 10 by any change?

**Comment by maxired** [comment]
@mcdonnelldean , I am gonna do the upgrade right now from windows 8.1
I am more a linux guy, but keep a Windows for this kind of issue ;)

**Comment by mcdonnelldean** [comment]
@maxired Awesome, if so will you give this a quick whirl and see if it works there? I need to get my VM's in order :D

**Comment by feranf** [comment]
@mcdonnelldean @maxired 
Regarding the firewall, I'm using Symantec firewall, but I disabled it completely so I'm pretty sure its not it.
I'll search and try windows 10 to test on and will let you know.
(I'll submit also an issue also in https://github.com/rjrodger/seneca)

**Comment by feranf** [comment]
@mcdonnelldean @maxired Just wanted to clarify that it happened using **both http and tcp** transports...
Any way, thanks a lot for your efforts :+1:

**Comment by thekemkid** [comment]
I'm seeing this issue on win10. and I allowed it through my firewall. I'm investigating :)

**Comment by thekemkid** [comment]
So far, I have disabled the firewall, and I'm still seeing the issue. Diving deeper.

**Comment by thekemkid** [comment]
Oh, and node v4.2.0 and seneca v7.1, and windows 10.

**Comment by mcdonnelldean** [comment]
@thekemkid Awesome work, can I leave this with you? @feranf No need to open the other issue, lets keep it all here at this point :D

**Comment by thekemkid** [comment]
I'll probably need some support when I get deeper into the internals, I'll talk through what I'm seeing on the gitter for this org?

**Comment by thekemkid** [comment]
Actually, scratch that, will talk in main seneca room! :)

**Comment by mcdonnelldean** [comment]
Perfect

**Comment by feranf** [comment]
@thekemkid Thanks, waiting for good news...
@mcdonnelldean @maxired thanks again, I'll cancel the other issue

**Comment by mcdonnelldean** [comment]
@feranf Awesome :D Cheers

**Comment by thekemkid** [comment]
Okay, this is working code:

```
function color() {
  this.add( 'color:red', function(args,done){
    done(null, {hex:'#FF0000'});
  })
}


var seneca = require('seneca');

seneca()
  .use(color)
  .listen({type:'tcp'}) //become a server

seneca()
  .client({type:'tcp', host:"localhost"}) //become a client
  .act('color:red', console.log)
```

**Comment by maxired** [comment]
@thekemkid , well done.

To clarify, I spoke too quickly and the code was indeed not working on windows 8 without the workaround provided by @thekemkid

**Comment by thekemkid** [comment]
No problem, I'll self assign this and send a pr to update the documentation

**Comment by feranf** [comment]
@thekemkid Great news, Its working now! Thanks a lot for the quick response :+1:

**Comment by mcdonnelldean** [comment]
@thekemkid Awesome stuff

@feranf Any other issues, please let us know.

**Comment by feranf** [comment]
@mcdonnelldean Thanks, appreciate this.
